main/cmds.go

```diff
@@ -79,6 +79,8 @@ func install(ctx *log.Context, h HandlerEnvironment, seqNum int) (string, error)
 }
 
 func migrateToMostRecentSequence(ctx *log.Context, h HandlerEnvironment, seqNum int) {
+	// First the config seq number is used for mrseq. If there is an error saving that, then... 
```

Comment:
"config seq number" looks like a description. What's the env variable name ? Also, please mention what component saves it.